modules/cluster-logging-collector-log-forward-cloudwatch.adoc

```diff
@@ -0,0 +1,276 @@
+$ oc logs -f busybox
+My life is my message
+My life is my message
+My life is my message
+...
+----
+
+You can also get the UUID of the project.
+[source,terminal]
+----
+$ oc get ns/app -ojson | jq .metadata.uid
+"794e1e1a-b9f5-4958-a190-e76a9b53d7bf"
+----
+
+In your `ClusterLogForwarder` custom resource (CR), the `all-logs` pipeline forwards `infrastructure`, `audit`, and `application` logs to the CloudWatch  output, `cw`:
+
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: ClusterLogForwarder
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  outputs:
+   - name: cw
+     type: cloudwatch
+     cloudwatch:
+       region: us-east-2
+       groupBy: groupType
+     secret:
+        name: cw-secret
+  pipelines:
+    - name: all-logs
+      inputRefs:
+        - infrastructure
+        - audit
+        - application
+      outputRefs:
+        - cw
+----
+
+In CloudWatch, each AWS region contains three levels of objects:
+
+* region
+** log group
+*** log stream
+**** log event
+
+
+The three log types in the `ClusterLogForwarding` CR create three log groups in AWS:
+
+[source,terminal]
+----
+$ aws --output json logs describe-log-groups | jq .logGroups[].logGroupName
+"mycluster-7977k.application"
+"mycluster-7977k.audit"
+"mycluster-7977k.infrastructure"
+----
+
+Each of these log groups contains log streams:
+
+[source,terminal]
+----
+$ aws --output json logs describe-log-streams --log-group-name mycluster-7977k.application | jq .logStreams[].logStreamName
+"kubernetes.var.log.containers.busybox_app_busybox-da085893053e20beddd6747acdbaf98e77c37718f85a7f6a4facf09ca195ad76.log"
+----
+
+[source,terminal]
+----
+$ aws --output json logs describe-log-streams --log-group-name mycluster-7977k.audit | jq .logStreams[].logStreamName
+"ip-10-0-131-228.us-east-2.compute.internal.k8s-audit.log"
+"ip-10-0-131-228.us-east-2.compute.internal.linux-audit.log"
+"ip-10-0-131-228.us-east-2.compute.internal.openshift-audit.log"
+...
+----
+
+[source,terminal]
+----
+$ aws --output json logs describe-log-streams --log-group-name mycluster-7977k.infrastructure | jq .logStreams[].logStreamName
+"ip-10-0-131-228.us-east-2.compute.internal.kubernetes.var.log.containers.apiserver-69f9fd9b58-zqzw5_openshift-oauth-apiserver_oauth-apiserver-453c5c4ee026fe20a6139ba6b1cdd1bed25989c905bf5ac5ca211b7cbb5c3d7b.log"
+"ip-10-0-131-228.us-east-2.compute.internal.kubernetes.var.log.containers.apiserver-797774f7c5-lftrx_openshift-apiserver_openshift-apiserver-ce51532df7d4e4d5f21c4f4be05f6575b93196336be0027067fd7d93d70f66a4.log"
+"ip-10-0-131-228.us-east-2.compute.internal.kubernetes.var.log.containers.apiserver-797774f7c5-lftrx_openshift-apiserver_openshift-apiserver-check-endpoints-82a9096b5931b5c3b1d6dc4b66113252da4a6472c9fff48623baee761911a9ef.log"
+...
+----
+
+Each log stream contains log events. Here, to see a log event from the `busybox` Pod, you specify a log stream from the `application` log group:
```

Comment:
`Here` indicates a slightly conversational tone, which is totally fine and appropriate per the IBM SG. If you wanted to cut word count and keep it a bit more formal, I think it reads equally well to say, 

`To see a log event from the busybox Pod, specify a log stream the application log group` 

Either phrasing is good though!